app/models/donor.rb

```diff
@@ -3,7 +3,7 @@ class Donor < ActiveRecord::Base
 
   validates :identification, presence: true, uniqueness: true
 
-  scope :search, -> (keyword) { where("name ILIKE ?", "%#{keyword}%") }
+  scope :search, -> (keyword) { where("name ILIKE ? OR  identification ILIKE ?", "%#{keyword}%", "%#{keyword}%") }
```

Comment:
You can use a "placeholder condition" here, so you don't have to repeat the keyword argument:

```
where("name ILIKE :keyword OR identification ILIKE :keyword", { keyword: "%#{keyword}%" })
```